content/en/releases/_index.md

```diff
@@ -22,6 +22,6 @@ More information in the [version skew policy](/releases/version-skew-policy/) do
 
 ## Upcoming Release
 
-Check out the [schedule](https://github.com/kubernetes/sig-release/tree/master/releases/release-{{< skew nextMinorVersion >}}) for the upcoming **{{< skew nextMinorVersion >}}** Kubernetes release!
+Check out the [schedule](https://github.com/kubernetes/sig-release/tree/master/releases/release-{{< skew currentVersionAddMinor 1 >}}) for the upcoming **{{< skew currentVersionAddMinor 1 >}}** Kubernetes release!
```

Comment:
I'm not sure about this change. What might be better is to have the `/releases/` section detect when it is served for an older release, and omit some of the text (including the “upcoming release” section).

Reply:
That won't work (Hugo doesn't let you do it), but the logic is right.

Reply:
@sftim I think that it would be better to open a new issue based on [the suggestion you gave](https://github.com/kubernetes/website/pull/34349#discussion_r899899399),
rather than trying to resolve within this PR.

Reply:
Or, deleting it manually for this time (possibly in this PR and other backporting PRs) could be one alternative option.